src/index.html

```diff
@@ -1,13 +1,416 @@
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+
+            <a href="#" class="logo">
+              <img
+                class="header__logo-image"
+                src="./images/bose/logo.png"
+                alt="bose logo"
+              >
+            </a>
+
+            <a href="tel:+1 234 5555-55-55" class="header__call icon icon--phone"></a>
+            <a href="#menu" class="icon icon--menu"></a>
+
+          </div>
+
+          <h1 class="header__title">
+            The world<br>shades.
+            <br>
+            Your music<br>shines.
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="top-actions menu__top">
+
+            <a href="#" class="header__logo">
+              <img
+                class="header__logo-image"
+                src="./images/bose/logo.png"
+                alt="bose logo"
+              >
+            </a>
+
+            <a href="#"></a>
+            <a href="#" class="icon icon--cross"></a>
+
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">home</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#recommended" class="menu__link">recommended</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#products" class="menu__link">products</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#why-bose" class="menu__link">why BOSE</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">contact us</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--phone"
+          >
+            +1 234 5555-55-55
+          </a>
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--action"
+          >
+            call to order
+          </a>
+
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section recommended" id="recommended">
+        <div class="container">
+          <h2 class="page__section-title">Recommended</h2>
+
+          <div class="grig grid--desktop">
+            <div class="recommended__item grid__item grid__item--desktop-1-4">
+              <div class="item">
+                <div class="item__photo-container">
+                  <img
+                    class="item__photo"
+                    src="./images/bose/img1.png"
+                    alt="Bose portable Smart speaker"
+                  >
```

Comment:
Checklist 8th item. Pictures in Gallery should increase on hover
![image](https://user-images.githubusercontent.com/37161315/170449839-5b0d79d3-83b4-4738-a06a-9fb3d1e53c0e.png)

Reply:
same here
![image](https://user-images.githubusercontent.com/37161315/170449888-7410a9f6-45d0-4846-ab1e-1fb9731ca07e.png)